README.md

```diff
@@ -1,6 +1,7 @@
 # Prometheus Operator
 [![Build Status](https://travis-ci.org/coreos/prometheus-operator.svg?branch=master)](https://travis-ci.org/coreos/prometheus-operator)
 [![Go Report Card](https://goreportcard.com/badge/coreos/prometheus-operator "Go Report Card")](https://goreportcard.com/report/coreos/prometheus-operator)
+[![Slack](https://img.shields.io/badge/join%20slack-%23prometheus--operator-brightgreen.svg)](http://slack.k8s.io/)
```

Comment:
should the link be https? Also, the link is to the general k8s slack, can we somehow link directly to the right channel?